backend/config/urls.py

```diff
@@ -6,17 +6,24 @@
 from django.views.generic import TemplateView
 from drf_spectacular.views import SpectacularAPIView, SpectacularSwaggerView
 
+home = TemplateView.as_view(
+    template_name="pages/home.html",
+    extra_context={
+        "api_url": settings.API_URL if settings.API_URL else "",
```

Comment:
Sorry, I wasn't very precise, settings will always have the values, there should be no need for a fallback.